packages/cursorless-engine/src/processTargets/ModifierStageFactoryImpl.ts

```diff
@@ -0,0 +1,134 @@
+import {
```

Comment:
I wonder if we want to move these factories into the composition root; one could argue they're really part of the composition root, but otoh there's something nice about having them close to all the things they instantiate.  Idk wdyt @auscompgeek @AndreasArvidsson ?